lib/doctrine.js

```diff
@@ -351,13 +351,46 @@
 
                     var ch;
                     var bracketDepth = 1;
+
                     // scan in the default value
                     while (index < last) {
                         ch = source.charCodeAt(index);
 
                         if (esutils.code.isWhiteSpace(ch)) {
-                            skipWhiteSpace(last);
-                            ch = source.charCodeAt(index);
+                            if (!insideString) {
+                                skipWhiteSpace(last);
+                                ch = source.charCodeAt(index);
+                            }
+                        }
+
+                        if (ch === 0x27 /* ''' */) {
+                            if (!insideString) {
+                                insideString = '\'';
+                            } else {
+                                if (insideString === '\'') {
+                                    insideString = false;
+                                }
+                            }
+                        }
+
+                        if (ch === 0x22 /* '"' */) {
+                            if (!insideString) {
+                                insideString = '"';
+                            } else {
+                                if (insideString === '"') {
+                                    insideString = false;
+                                }
+                            }
+                        }
+
+                        if (ch === 0x60 /* '`' */) {
```

Comment:
I haven't found anything about backticks (though nothing in the JSDoc docs seems to say anything about single or double quotes either). In fact, the example they give for a param with a default value that's a string is:

```js
/**
 * @param {string} [somebody=John Doe] - Somebody's name.
 */
function sayHello(somebody) {
    if (!somebody) {
        somebody = 'John Doe';
    }
    alert('Hello ' + somebody);
}
```

I'll remove the backticks. I like working on these JSDoc issues because I'm not super familiar beyond the basics and I learn a lot every time I dig in 😄